lib/cds_importer/entity_mapper/measure_mapper.rb

```diff
@@ -30,13 +30,11 @@ class MeasureMapper < BaseMapper
       self.entity_mapping_keys_to_parse = mapping_keys_to_parse.freeze
 
       before_oplog_inserts do |xml_node|
-        if xml_node['sid'].blank?
```

Comment:
We've been warning about this and it's never happened. I thought this was defensive at the time and I do now. If the Measure sid is missing we won't soft-delete the associations or even be able to update the Measure record and it's other associations.